### PR TITLE
Minor grammer/spelling changes

### PR DESCRIPTION
**Pull request by johnma14**

### CODE REVIEW

File: content/docs/concepts/traffic-management/index.md [hunk 1]
```diff
@@ -93,7 +93,7 @@ etc.). Platform-specific adapters are responsible for populating the
 internal model representation with various fields from the metadata found
 in the platform.
 
-Istio introduces the concept of a _service version_, which is a finer-grained
+Istio introduces the concept of a _service version_, which is a fine-grained
```
Comment (venilnoronha) [inline content/docs/concepts/traffic-management/index.md]: I'd leave it as _fine-grained_.

File: content/docs/concepts/traffic-management/index.md [hunk 2]
```diff
@@ -133,7 +133,7 @@ Istio does not provide a DNS. Applications can try to resolve the
 FQDN using the DNS service present in the underlying platform (`kube-dns`,
 `mesos-dns`, etc.).
 
-### Ingress and egress
+### Ingress and Egress
```
Comment (geeknoid) [inline content/docs/concepts/traffic-management/index.md]: Should we lowercase e, we only capitalize the first letter of headings.

**Comment by geeknoid** [comment]
We only capitalize the first letter of headings, except for proper names and acronyms.

Page titles do use title case however.

**Comment by geeknoid** [comment]
BTW, you may wish to look at https://preliminary.istio.io/about/contribute/style-guide/

File: content/docs/concepts/traffic-management/index.md (hunk 2 above)
Comment (johnma14) [reply content/docs/concepts/traffic-management/index.md]: I read the style guide and I agree.

File: content/docs/concepts/traffic-management/index.md [hunk 3]
```diff
@@ -194,7 +194,7 @@ Services can actively shed load by responding with an HTTP 503 to a health
 check. In such an event, the service instance will be immediately removed
 from the caller's load balancing pool.
 
-## Handling failures
+## Handling Failures
```
Comment (johnma14) [inline content/docs/concepts/traffic-management/index.md]: Agree, this doesn't need to be capitalized.

**Pull request closed without merging**